db/migrate/20170923230002_create_posts.rb

```diff
@@ -2,8 +2,8 @@ class CreatePosts < ActiveRecord::Migration[5.1]
   def change
     create_table :posts do |t|
       t.text :body
-      t.integer :user_id, null: false
-      t.integer :receiver_id, null: false
+      t.references :author, null: false
```

Comment:
I find using references much more explicit when creating foreign keys. Rails does a great job of this out the box too. Any objections to this approach?

Reply:
👍